src/ja/server/database/types/work_machine.py

```diff
@@ -64,14 +66,20 @@ class WorkMachineConnectionDetails:
     # TODO: this class is incomplete
 
 
+class WorkMachineState(Enum):
+    ONLINE = 0
```

Comment:
not a problem, just curious: why 0, 10 and 20 instead of 0,1,2?

Reply:
I wasn't 100% if we will need more states.
I figured that if I leave some space in between states it would be easier to add more later.